tests/test_repodiff.py

```diff
@@ -0,0 +1,23 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: MIT
+import json
+from collections import namedtuple
+from collections.abc import Generator
+
+from _pytest.capture import CaptureFixture
+
+from openqabot.repodiff import RepoDiff
+
+_namespace = namedtuple("Namespace", ("dry", "fake_data", "repo_a", "repo_b"))
+
+
+def test_repodiff(capsys: Generator[CaptureFixture[str]]) -> None:
```

Comment:
Also this is quite new to me. do you really need `Generator` here?
According to https://docs.pytest.org/en/7.1.x/reference/reference.html?highlight=capsys#pytest.capture.capsys the following would work as well, no?

```import pytest

def test_repodiff(capsys: pytest.CaptureFixture[str]) -> None:
```